cirq/protocols/resolve_parameters_test.py

```diff
@@ -58,6 +58,18 @@ def _resolve_parameters_(self, resolver: ParamResolver):
     assert cirq.resolve_parameters(SimpleParameterSwitch('a'), d).parameter == 0
     assert cirq.resolve_parameters(sympy.Symbol('a'), r) == 0
 
+    a, b, c = tuple(sympy.Symbol(l) for l in 'abc')
+    x, y, z = 0, 4, 7
+    resolver = {a: x, b: y, c: z}
+    assert cirq.resolve_parameters((a, b, c), resolver) == (x, y, z)
```

Comment:
Check the empty list corner case, and also that a list with no parameters is returned unchanged.